woocommerce/woocommerce-ios
# JCP: Install Jetpack steps UI

Part of #5365 

# Description
This PR adds new UI for Install Jetpack steps. The screen is displayed when tapping "Get Started" on Install Jetpack intro view.

# Changes
- Added a container view `JetpackInstallView` that switches between showing the intro and steps views.
- Added `JetpackInstallStep` enum for the steps needed for installing Jetpack.
- Added UI for `JetpackInstallStepsView`. For demo purpose the view is displaying activation step by default. This should be updated when API integration is added.

# Screenshots
| Light mode | Dark mode |
| ----- | ----- |
| <img src="https://user-images.githubusercontent.com/5533851/142597893-ddcd24ea-05e2-475b-9617-328122199532.png" width=320 /> | <img src="https://user-images.githubusercontent.com/5533851/142597954-f4ad4c0c-2561-4676-b7ee-b7bea004629f.png" width=320 /> |


# Testing
1. Make sure that `.jetpackConnectionPackageSupport` feature flag is on.
2. Tap Jetpack Benefits banner on My Store tab, then tap "Install Jetpack".
3. On Jetpack Install screen, tap "Get Started". Notice that you're navigated to a new screen with steps for the installation.
4. Extra: change `currentStep` on `JetpackInstallStepsView.swift` to other values to see if the view reflects correct state.

Update release notes:
